isis/tests/FunctionalTestsHimos.cpp

```diff
@@ -62,7 +62,7 @@ TEST_F(MroHiriseCube, FunctionalTestHimosDefault) {
     StopTime                  = 2006-09-29T15:16:35.036
     SpacecraftClockStopCount  = 844010213:55196
     IncidenceAngle            = 59.687930340662 <DEG>
-    EmissionAngle             = 0.091672512443932 <DEG>
+    EmissionAngle             = 0.091672512439956 <DEG>
```

Comment:
This is now failing for linux as seen in the nightly builds:
https://jenkins.prod-asc.chs.usgs.gov:8443/job/ISIS/view/change-requests/job/PR-5082/

Reply:
I am wondering if we should avoid checking against the label this way. It does not allow us to compare within a certain decimal range.

Reply:
I will work on getting these switched to individual asserts.